package.json

```diff
@@ -31,5 +31,8 @@
             "react-app",
             "react-app/jest"
         ]
+    },
+    "scripts": {
+        "test": "sudo ./tests/run-unit-tests.sh"
```

Comment:
Never `sudo` things. Also, I don't think you should be adding this to `package.json`

Reply:
Sorry, that was for my convenience, it will not be included in the final PR, I also got permissions problem with docker which I postponed solving until I have more time :)